ctapipe/image/toymodel.py

```diff
@@ -369,3 +399,88 @@ def pdf(self, x, y):
         r = np.sqrt((x - self.x) ** 2 + (y - self.y) ** 2)
 
         return norm(self.radius.to_value(u.m), self.sigma.to_value(u.m)).pdf(r)
+
+
+class SkewedGaussianLaplace(ImageModel):
```

Comment:
Is there a physical motivation for this? Does it fit better?

Reply:
We thought this could better estimate the width. But, after some tests, I do not think it works better than the 2D Skewed Gaussian so I think I could remove it